spirekey/src/components/AccountDetail/AccountDetail.tsx

```diff
@@ -1,42 +1,174 @@
+'use client';
+
+import {
+  Button,
+  Grid,
+  Heading,
+  NumberField,
+  Stack,
+  Text,
+  TextField,
+} from '@kadena/react-ui';
+import { useParams, useRouter } from 'next/navigation';
+
 import { useAccounts } from '@/context/AccountsContext';
 import { useNotifications } from '@/context/shared/NotificationsContext';
-import { calculateBalancePercentage } from '@/utils/balance';
-import { Box, GridItem, Heading, Link, Stack } from '@kadena/react-ui';
+
+import {
+  objectParameterValue,
+  searchParamsToString,
+} from '@/utils/searchParameters';
+import { updateAccount } from '@/utils/updateAccount';
+import { Account } from '@kadena-spirekey/types';
```

Comment:
should this type have this type keyword? or is typescript smart enough to shake it?

Reply:
I don't think there's anything but types in the package, but it's good practice to add the type keyword (so I did that now 😸 )